.circleci/config.yml

```diff
@@ -182,10 +181,15 @@ gn-build-steps: &gn-build-steps
         name: GN gen
         command: |
           cd src
-          SCCACHE_PATH="$PWD/libchromiumcontent/tools/sccache/aad2120/linux/sccache"
-          echo 'export SCCACHE_WRAPPER="'"$SCCACHE_PATH"'"' >> $BASH_ENV
+          SCCACHE_BUCKET="electronjs-sccache"
+          SCCACHE_PATH="$PWD/electron/external_binaries/sccache"
+          SCCACHE_TWO_TIER=true
```

Comment:
We probably don't want `TWO_TIER` on CI, it won't push things to S3 with two tier enabled

Reply:
SCCACHE_BUCKET gets set on the CI UI, not in the config (easier if we ever need to move it).